Withings/OAuth1/DefaultTokenManager.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Withings.API.Portable.OAuth1
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    class DefaultTokenManager : ITokenManager //from ITokenManager
```

Comment:
Also, a lot of the nitty-gritty of making OAuth 1.0a work is going to be already done for you internally using the `AsyncOAuth` library. In fact, you likely could run the entire connection flow right through that w/o this library just to get connected. Study those docs and examples well.